codalab/lib/codalab_manager.py

```diff
@@ -184,6 +190,11 @@ def init_config(self, dry_run=False):
                     'verbose': 1,
                     'dispatch_command': "python $CODALAB_CLI/scripts/dispatch-q.py",
                 }
+            },
+            'logging': {
+                'file_path': "$CODALAB_HOME/codalab.log",
+                'console_level': "INFO",
+                'file_level': "DEBUG",
```

Comment:
DEBUG would probably create too much crap in the log file. DEBUG is meant for debugging, not production use. I would default to INFO.